### ISSUE TITLE
iOS version

### ISSUE THREAD

**Issue by prusnak** [minibits-cash/minibits_wallet#114]
Is the iOS version planned? Technically, this should be possible, since the project is written in React Native.

**Comment by minibits-cash** [comment]
Yes sometimes next year. Will need to look after some help to get it done.

**Comment by prusnak** [comment]
Great!

I tried to build from the source by myself and I faild. The whole Xcode thing is a total mess :-/

**Comment by minibits-cash** [comment]
Agree. Some of the libs I use require tweaks to xcode project, needs to be done one by one. No showstoppers, I chose the deps so that they do not have blocking issues on iOS.

Matter of strong will and enough time to kick it until it works. 

Had a smart young student helping me spring and summer but he got admitted to fairly serious math uni from this sept and is out of any free time as of now...

Will likely need to look for another one, not sure anybody will just volunteer for a task like this. Not a fan of iOS from dev POV.

**Comment by prusnak** [comment]
Have you thought about introducing [Expo](https://expo.dev) to the project or you want to keep it lean?
It is an extra dependency, but simplifies cross-platform development a lot.

**Comment by minibits-cash** [comment]
Did one previous project using Expo and decided to drop it from this one. Staying lean was one motivation, step out of the pace of mandatory upgrades was the second and did not want to invest time to learn their new build tooling. As well, somehow could not get rid of bundled facebook trackers before, but they may be fixed that one, as they decomposed many libs to separate packages over time.

**Comment by minibits-cash** [comment]
I dare to close this one now, when Minibits made it to the https://freedomstore.io/
Thank you to push me to jump in!
